web/packages/teleterm/src/ui/Vnet/DiagnosticsAlert.tsx

```diff
@@ -69,9 +73,34 @@ export const DiagnosticsAlert = () => {
         title: 'Log in to a cluster to see the full report',
       }
     : {};
-  const openReport = () => {
-    // TODO(ravicious): Open report doc.
-  };
+  const openReport = !rootClusterUri
```

Comment:
I'd probably return early in the function if `!rootClusterUri` but this works too.

Reply:
how did I not think of this 😂🤦